mitodl/ocw-to-hugo
# Create page for each resource

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #335 
Related to https://github.com/mitodl/ocw-studio/issues/508

#### What's this PR do?
Creates resource pages for all files and youtube media. PDF pages are now links to `/resources/...`

#### How should this be manually tested?
Diffing this vs master won't work since there are many changes as well as changes in file locations. Instead I would just check a few things in a few courses:
 - Convert `18-06-linear-algebra-spring-2010` using this PR
 - Start ocw-hugo-themes pointing to the outputted data. Use this branch: https://github.com/mitodl/ocw-hugo-themes/pull/172
 - Go to http://localhost:3000/pages/video-lectures/ and click on some video links. You should see something very similar to what's on production: https://ocwnext.odl.mit.edu/courses/18-06-linear-algebra-spring-2010/pages/video-lectures/. Videos should work fine. I've noticed captioning doesn't work but that's a separate issue https://github.com/mitodl/ocw-data-parser/issues/155
 - Go to http://localhost:3000/resources/18-06classroom/. You should see a link where you can download an image file of a classroom. Note that the image is linked on http://localhost:3000/pages/instructor-insights/ but it won't show up due to #352
 - Go to http://localhost:3000/resources/mit18_06s10_pset1_s10_soln/. You should see a PDF render properly. If you go to /pages/assignments and click the first link that says "PDF", it should go to this page.
